validator/sawtooth_validator/journal/consensus/consensus.py

```diff
@@ -41,41 +41,48 @@ def __init__(self, block_cache, state_view):
         pass
 
     @abstractmethod
-    def initialize_block(self, block):
+    def initialize_block(self, block_header):
         """Do initialization necessary for the consensus to claim a block,
         this may include initiating voting activities, starting proof of work
-        hash generation, or create a PoET wait timer.
+        hash generation, or create a PoET wait timer. The block
+        is represented by a block_header since the full block will not be
+        built until the block is finalized.
 
         Args:
-            block (Block): the block to initialize.
+            block_header (BlockHeader): the BlockHeader to initialize.
         Returns:
-            consensus: the serialized consensus data for the block header, this
-            can be temporary state data or None.
+            Boolean: True if the candidate block should be built. False if
+            no candidate should be built.
         """
         pass
 
     @abstractmethod
-    def check_publish_block(self, block):
-        """Check if a candidate block is ready to be claimed.
+    def check_publish_block(self, block_header):
+        """Check if a candidate block is ready to be claimed. The block
+        is represented by a block_header since the full block will not be
+        built until the block is finalized.
 
         Args:
-            block (Block): the block to be checked if it should be claimed
+            block_header (BlockHeader): the block_header to be checked if it
+            should be claimed
         Returns:
-            Boolean: True if the candidate block should be claimed.
+            Boolean: True if the candidate block should be claimed. False if
+            the block is not ready to be claimed.
         """
         pass
 
     @abstractmethod
-    def finalize_block(self, block):
-        """Finalize a block to be claimed. Provide any signatures and
-        data updates that need to be applied to the block before it is
-        signed and broadcast to the network.
+    def finalize_block(self, block_header):
+        """Finalize a block to be claimed. Update the
+        Block.block_header.consensus field with any data this consensus
+        BlockVerifier.
 
         Args:
-            block (Block): The candidate block that needs to be finalized
-             by the consensus
+            block_header (BlockHeader): The candidate block that needs to be
+            finalized.
         Returns:
-            consensus: The consensus data to store on the block.
+            Boolean: True if the candidate block good, False if the block
```

Comment:
"...candidate block should be finalized, ..."?
"...candidate block is good, ...."?